stores/metadata.go

```diff
@@ -1787,7 +1782,7 @@ func (s *SQLStore) dirID(tx *gorm.DB, dirPath string) (uint, error) {
 
 func makeDirsForPath(tx *gorm.DB, path string) (uint, error) {
 	// Create root dir.
-	dirID := uint(sql.DirectoriesRootID)
+	dirID := uint(isql.DirectoriesRootID)
```

Comment:
should `DirectoriesRootID` be defined in the stores package rather than internal

Reply:
Can't do that because it's used in the new `MakeDirsForPath` and those can't import `stores` due to a circular dependency. Good news is that `makeDirsForPath` will be gone soon anyway so the `isql` shouldn't be necessary here anymore.